### DISCUSSION TITLE
Change file position

### DISCUSSION THREAD

**Discussion by Udaymakxenia** [pschatzmann/arduino-audiokit#74] [Q&A] [answered]
Hi Phil , I wanted to control the file position from which audio is played from the SD card. I am using AudioPlayer.h. 
File position like when I play an mp3 file it starts from the beginning , I want to control this starting position .

**Comment by pschatzmann** [comment, marked as answer]
Just get the actual Stream: This can be casted to a file on which you can do any file operations like lseek...

ps you posted this to the wrong project. The audio player is part of the AudioTools and you need to check the class documentation that you can find there!

As an alternative you can implement your own AudioSource (or by using the AudioSourceCallback class)

**Reply by Udaymakxenia** [reply]
Thanks Phil . Got it working.
